crates/forge_loader/src/manifest.rs

```diff
@@ -86,11 +157,28 @@ pub struct ForgeModules<'a> {
     scheduled_triggers: Vec<ScheduledTrigger<'a>>,
     #[serde(rename = "consumer", default, borrow)]
     pub consumers: Vec<Consumer<'a>>,
+    #[serde(rename = "compass:dataProvider", default, borrow)]
+    pub data_provider: Vec<DataProvider<'a>>,
+    #[serde(rename = "jira:customField", default, borrow)]
+    pub custom_field: Vec<CustomField<'a>>,
+    #[serde(rename = "jira:uiModificatons", default, borrow)]
+    pub ui_modifications: Vec<UiModificatons<'a>>,
+    #[serde(rename = "jira:workflowValidator", default, borrow)]
+    pub workflow_validator: Vec<WorkflowValidator<'a>>,
     #[serde(rename = "jira:workflowPostFunction", default, borrow)]
-    workflow_post_functions: Vec<WorkflowPostFunction<'a>>,
+    pub workflow_post_function: Vec<WorkflowPostFunction<'a>>,
```

Comment:
<!--
 Copyright 2024 Joshua Wong.
 SPDX-License-Identifier: Apache-2.0
-->

You're missing these jira modules:

- [`jira:globalPage`](https://developer.atlassian.com/platform/forge/manifest-reference/modules/jira-global-page/#jira-global-page)
- [`jira:issuePanel`](https://developer.atlassian.com/platform/forge/manifest-reference/modules/jira-issue-panel/#properties)
- [`jira:issueAction`](https://developer.atlassian.com/platform/forge/manifest-reference/modules/jira-issue-action/#properties)
- [`jira:issueContext`](https://developer.atlassian.com/platform/forge/manifest-reference/modules/jira-issue-context/#properties)
- [`jira:issuePanel`](https://developer.atlassian.com/platform/forge/manifest-reference/modules/jira-issue-panel/#properties)
- [`jira:issueViewBackgroundScript`](https://developer.atlassian.com/platform/forge/manifest-reference/modules/jira-issue-view-background-script/#properties)
- [`jira:projectSettingsPage`](https://developer.atlassian.com/platform/forge/manifest-reference/modules/jira-project-settings-page/#properties)
- [`jira:projectPage`](https://developer.atlassian.com/platform/forge/manifest-reference/modules/jira-project-page/#properties)
- [`jira:projectSettingsPage`](https://developer.atlassian.com/platform/forge/manifest-reference/modules/jira-project-settings-page/#properties)
- [`jira:issueActivity`](https://developer.atlassian.com/platform/forge/manifest-reference/modules/jira-issue-activity/#properties)
- [`jira:jqlFunction`](https://developer.atlassian.com/platform/forge/manifest-reference/modules/jql-function/#properties)
- [`jira:dashboardBackgroundScript`](https://developer.atlassian.com/platform/forge/manifest-reference/modules/jira-dashboard-background-script/#properties)
- [`jira:dashboardGadget`](https://developer.atlassian.com/platform/forge/manifest-reference/modules/jira-dashboard-gadget/#properties)
  Note that there's an optional invokable function at the `edit` property for this one.
- [`customFieldType`](https://developer.atlassian.com/platform/forge/manifest-reference/modules/jira-custom-field-type/#properties)
  Note that there's a *lot* of invokable functions on this module, so read through them carefully. The only non invokable one is the `searchSuggestions` function.

Reply:
Done